eth-rpc-adapter/src/transports/server-transport.ts

```diff
@@ -34,6 +35,20 @@ export abstract class ServerTransport {
       console.warn('transport method called without a router configured.'); // tslint:disable-line
       throw new Error('No router configured');
     }
+    // Get datadog span from the context
+    const span = process.env.EXTENSIVE_DD_INSTRUMENTATION == 'true' ? tracer.scope().active() : null;
+    // Initialize datadog span tags
+    const spanTags = span
+      ? {
+          body: {
+            method,
+            params
+          },
+          enterTime: Date.now(),
```

Comment:
should use `performance.now()` for benchmarking purpose

Reply:
Fixed!!
Thanks 👍